src/main/java/core/basesyntax/serviceimpl/FileReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReaderService {
+
+    @Override
+    public List<String> getDataFromFile(File file) {
```

Comment:
The method parameter should be a string representing the file path, rather than a File object, to avoid hardcoding the file path within the service methods.